### PR TITLE
chore: Update translations

### PR DESCRIPTION
**Pull request by RuffleBuild**
New Crowdin pull request with translations

### CODE REVIEW

**Comment by torokati44** [comment]
Could you please review, @sombraguerrero?

**Comment by torokati44** [comment]
Hmh, checks are not running again, @Dinnerbone ... :thinking:
Does rebasing work differently in this regard from creating the PR...?

**Comment by Dinnerbone** [comment]
There was a github outage last night, maybe?

**Comment by torokati44** [comment]
Hmh... Re-ran the job from last night, hence the force push, but still no checks running. *le sigh*

**Comment by Dinnerbone** [comment]
Try another pr to check? There's nothing special about this PR, especially if you are the one to hit rebase.

**Comment by torokati44** [comment]
> especially if you are the one to hit rebase.

nah, 'twas the GHA workflow. no rebase button if it's already on top of HEAD unforty

**Comment by Dinnerbone** [comment]
You said you force pushed...?

**Comment by torokati44** [comment]
https://github.com/ruffle-rs/ruffle/actions/runs/9981469330/job/27618391177#step:4:392

**Comment by torokati44** [comment]
> Could you please review, @sombraguerrero?

I meant the one Spanish translation only, of course... MT/AI stuff didn't convince me that it's right.

> You said you force pushed...?

I pressed "Re-run" on the same job that ran yesterday evening. It usually works to update the contents of translations from CrowdIn after hotfixing/undoing something.

**Comment by Dinnerbone** [comment]
I guess the push itself needs to happen with the token, not just making the PR?

**Comment by torokati44** [comment]
maybe!

**Comment by Dinnerbone** [comment]
The crowdin action uses `git push`, so I guess we need to find a way to change what credentials *that* uses

File: web/packages/core/texts/tr-TR/messages.ftl [hunk 1]
```diff
@@ -10,9 +10,20 @@ update-ruffle = Ruffle'ı Güncelle
...
+enable-hardware-acceleration-link = SSS - Chrome Donanım Hızlandırma
 view-error-details = Hata Ayrıntılarını Görüntüle
 open-in-new-tab = Yeni sekmede aç
 click-to-unmute = Sesi açmak için tıklayın
+clipboard-message-title = Ruffle'da kopyalama ve yapıştırma
+clipboard-message-description =
+    { $variant ->
+    *[unsupported] Tarayıcınız tam panoya erişimi desteklemiyor,
+    [access-denied] Pano erişimi reddedildi,
+    } ancak pano yerine her zaman bu kısayolları kullanabilirsiniz:
```
Comment (torokati44) [inline web/packages/core/texts/tr-TR/messages.ftl:23]: "instead of the clipboard"...?

**Comment by torokati44** [comment]
Eh, even if the Spanish and Turkish translations may not be the best, they can most likely be understood. They can always be fixed later.

**Pull request merged**